orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/ExecutionRunnerSupport.java

```diff
@@ -68,6 +63,7 @@ public ExecutionRunnerSupport(Collection<StageDefinitionBuilder> stageDefinition
         task.setId(taskDef.getId());
         task.setName(taskDef.getName());
         task.setStatus(NOT_STARTED);
+        task.setImplementingClass(taskDef.getImplementingClass());
```

Comment:
The original behavior was looking up task implementations by `name` which doesn't really work in practice ... going by `implementingClass` seems a little more consistent.